modules/op-about-pipelinerun.adoc

```diff
@@ -36,9 +36,13 @@ spec:
           requests:
             storage: 500Mi
 ----
-<1> PipelineRun API version `v1beta1`.
+<1> Pipeline run API version `v1beta1`.
 <2> Specifies the type of Kubernetes object. In this example, `PipelineRun`.
```

Comment:
```suggestion
<2> The type of Kubernetes object. In this example, `PipelineRun`.
```

Reply:
Parallelism: I'm suggesting this noun phrase so that it resembles the other callouts. Unlike the other callouts, this one begins with an article. It would be preferable for all the callouts to follow the same format, either with or without articles.